libraries/core/classes/PipelineRequest/index.js

```diff
@@ -62,6 +63,17 @@ class PipelineRequest extends Request {
     return this;
   }
 
+  /**
+   * Sets a flag to suppress errors.
+   * When true, no EVENT_PIPELINE_ERROR would be triggered.
+   * @param {bool} value Value.
+   * @return {PipelineRequest}
+   */
+  setSuppressErrors(value) {
+    this.suppressErrors = value;
```

Comment:
I think `suppressErrors` is misleading. The `PipelineRequest` object will still reject when an error occurs and there is still a handling necessary.

Additionally a 3rd party developer doesn't know what the triggering of `EVENT_PIPELINE_ERROR` means. So a more neutral explanation would be better e.g. `When true, the default error handling would be deactivated.`

Reply:
In order to ship our things on time I don't want to argue about naming. Please provide me the name you'd be happy to accept and I'd rename all the things.

Reply:
I've talked with our colleague from the platform again, and he was ok with the naming. So i'll accept it.